plugins/powerpaste.md

```diff
@@ -169,21 +172,11 @@ tinymce.init({
 
 {% include configuration/smart-paste.md %}
 
-## Toolbar Buttons
-
-#### pastetext
-
-This button allows you to toggle paste as plain text mode on and off. When in plain text mode, all rich content is converted into plain text.
-
-Example {{site.productname}} Configuration:
+{% assign altplugincode = "paste" %}
+{% include misc/plugin-toolbar-button-id-boilerplate.md %}
 
-```js
-tinymce.init({
-  selector: 'textarea',
-  plugins: 'powerpaste',
-  toolbar: 'pastetext'
-});
-```
+{% assign altplugincode = "paste" %}
+{% include misc/plugin-menu-item-id-boilerplate.md %}
 
 ## Menu Items
```

Comment:
Menu items is duplicated here

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1675/commits/9189463c6115d08e4d00e072ea557288f9416116